compatible/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.seata.spring.annotation;
+
+import io.seata.rm.RMClient;
+import io.seata.tm.TMClient;
+import org.apache.seata.common.util.StringUtils;
+import org.apache.seata.tm.api.FailureHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.apache.seata.common.DefaultValues.DEFAULT_TX_GROUP;
+import static org.apache.seata.common.DefaultValues.DEFAULT_TX_GROUP_OLD;
+
+public class GlobalTransactionScanner extends org.apache.seata.spring.annotation.GlobalTransactionScanner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GlobalTransactionScanner.class);
+
+    public GlobalTransactionScanner(String txServiceGroup) {
+        super(txServiceGroup);
+    }
+
+    public GlobalTransactionScanner(String txServiceGroup, int mode) {
+        super(txServiceGroup, mode);
+    }
+
+    public GlobalTransactionScanner(String applicationId, String txServiceGroup) {
+        super(applicationId, txServiceGroup);
+    }
+
+    public GlobalTransactionScanner(String applicationId, String txServiceGroup, int mode) {
+        super(applicationId, txServiceGroup, mode);
+    }
+
+    public GlobalTransactionScanner(String applicationId, String txServiceGroup, FailureHandler failureHandlerHook) {
+        super(applicationId, txServiceGroup, failureHandlerHook);
+    }
+
+    public GlobalTransactionScanner(String applicationId, String txServiceGroup, int mode, FailureHandler failureHandlerHook) {
```

Comment:
the origin `FailureHandler` is io.seata.tm.api.FailureHandler

Reply:
i will fix it